vyxal/elements.py

```diff
@@ -445,7 +445,7 @@ def process_element(
     "kṘ": process_element('"IVXLCDM"', 0),
     "k•": process_element('["qwertyuiop","asdfghjkl","zxcvbnm"]', 0),
     "¨P": process_element(
-        'in = pop(stack, 2, ctx) ; stack.append(eval(f"bs({in[1]},"html.parser").{in[0]}"))',
+        'in = pop(stack, 2, ctx) ; stack.append(str(eval(f"bs({in[1]},"html.parser").{in[0]}")))',
```

Comment:
Just converting to string returns the original HTML with slightly less newlines/somewhat neater formatting. It doesn't return any parsed information at all. You would need to return a list of all items in the HTML document.

Reply:
Now returns list with actual parse data.